src/swagger.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
This gets compiled by the swagger build script. This supplies it with the @SWG\Info, and at the moment the @SWG\Tag for 'release'. This then gets compiled (along with anything else) into swagger.json in wherever specified when running swagger